index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+  <title>Radium - A React Component Styling Library</title>
+  <meta name="description" content="A powerful set of tools for managing inline styles on React elements" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <!-- Fonts  -->
+  <link href="https://formidable.com/open-source/fonts.css" rel="stylesheet" type="text/css">
+  <style type="text/css">
+    body {
+      background-color: #e7e5e3;
+      color: #242121;
+      margin: 0;
+      box-sizing: border-box;
+      padding: 20px;
+      font-family: "akkurat", "Inconsolata", monospace;
+      font-size: 24px;
+      line-height: 1.4;
+      text-align: center;
+    }
+    a:link {
+      color: #c43a31;
+      font-weight: normal;
+      text-decoration: none;
+      transition: color 5s cubic-bezier(0.22, 0.61, 0.36, 1);
+    }
+    a:visited {
+      color: #cd5244;
+      border-bottom: 1px dotted #d1c7bc;
+    }
+    a:hover, a:focus {
+      cursor: "pointer";
+      color: #e58c7d;
+      transition: color 200ms cubic-bezier(0.22, 0.61, 0.36, 1);
+    }
+  </style>
+  <script>
+    var url = window.location.href;
+    if (url.indexOf("stack.formidable.com/radium") > -1) {
```

Comment:
Great question. I wrote this for victory awhile ago and can't remember. I can't imagine why it'd be needed here. Do you want to give it a go without the check?